### DISCUSSION TITLE
Global input style

### DISCUSSION THREAD

**Discussion by gius** [mantinedev/mantine#1696] [Q&A] [answered]
What is the recommended way to change the style (e.g., border color, font, etc.) of all inputs (TextInput, Select, DatePicker...)? Is there a shared theme property that is reused?

A similar question would be how to style Labels of all inputs.

Thank you!

**Comment by rtivital** [comment, marked as answer]
From version 5.0 – https://v5.mantine.dev/styles/styles-api/#inputs-styles-api-on-mantineprovider
In any version below 5.0 it is not supported.
